src/pages/assetGroup/AssetGroup.jsx

```diff
@@ -57,6 +58,10 @@ export default function AssetGroup() {
     a => get(a, 'guid'),
   );
 
+  const sightingCreator = data?.creator;
+  const creatorName = sightingCreator?.full_name || 'Unnamed user';
+  const creatorUrl = `/users/${sightingCreator?.guid}`;
```

Comment:
This link will be wonky if sightingCreator doesn't exist, and the link looks like it would still happen below.

Reply:
adjustment made below